site/en/docs/devtools/recorder/index.md

```diff
@@ -160,6 +168,42 @@ There are options to add and remove steps too. This is useful if you want to add
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/zeK91O21jvc5OD9HuovR.png", alt="The new step for coffee checkout has been updated with the aforementioned details.", width="800", height="663" %}
 6. [Replay](#replay) the flow now to see the changes.
 
+## Share user flows {: #share-flows }
+
+You can export and import user flows. This is useful for bug reporting because you can share an exact record of the steps that reproduce a bug.
+
+### Export a user flow {: #export-flows }
+
+To export a user flow:
+
+1. Open the user flow you want to export.
+2. Click on the **Export**<span class="material-icons">file_download</span> button at the top of the **Recorder** panel.
+    {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/LJJqpZ0vOEdp6rdDLXmP.png", alt="Download format options", width="800", height="490" %}
+ 1. Select one of the following formats from the dropdown list:
+    - **Export as a JSON file**. Download the recording as a JSON file.
+    - **Export as a @puppeteer/replay script**. Download the recording as a [Puppeteer Replay](https://github.com/puppeteer/replay) script.
+    - **Export as a Puppeteer script**. Download the recording as a [Puppeteer](https://pptr.dev/) script.
+1. Save the file.
+
+{% Aside 'gotchas' %}
+The [@puppeteer/replay](https://github.com/puppeteer/replay) is a library built on top of [Puppeteer](https://pptr.dev/). It is also a command line tool, so you can replay JSON files with it too.
+{% endAside %}
+
+You can do the following with each export option:
+
+- **JSON**. Edit the human-readable JSON object and [import](#import-flows) the JSON file back to the **Recorder**.
+- **@puppeteer/replay**. Replay the script with the [Puppeteer Replay](https://github.com/puppeteer/replay) library. Additionally, you can still edit the script because the steps remain a JSON object. The script is shorter as a result.
+- **Puppeteer** script. Replay the script with [Puppeteer](https://pptr.dev/). In this script, the steps are in JavaScript and you can't import them back into the **Recorder**. 
```

Comment:
```suggestion
- **Puppeteer** script. Replay the script with [Puppeteer](https://pptr.dev/). Since the steps are converted to JavaScript, it allow you to have more fine-grained customization (e.g. export to pdf, integration with Lighthouse, etc). One caveat, you can't import them back into the **Recorder**. 
```

Reply:
Thanks for the suggestion.
FYI:
- e.g. -> for example. See "e.g." in the [style guide](https://developers.google.com/style/word-list#letter-e)
- Don't use parentheses if you can avoid it. See [style guide](https://developers.google.com/style/parentheses)